projects/packages/videopress/src/class-initializer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * The initializer class for the videopress package
+ *
+ * @package automattic/jetpack-videopress
+ */
+
+namespace Automattic\Jetpack\VideoPress;
+
+/**
+ * Initialized the VideoPress package
+ */
+class Initializer {
+
+	/**
+	 * Invoke this method to initialize the VideoPress package
+	 *
+	 * @return void
+	 */
+	public static function init() {
+		if ( ! did_action( 'videopress_init' ) ) {
```

Comment:
Have you looked at the possibility of using Config? It gives you the security of knowing that the code was initialized only once, plus it opens the way for the package to be used with other plugins too.

Reply:
Yes, I added this comment in the description ;) 

I tried to initialize the package via Config::ensure, but Jetpack initialized the modules too late for that, and we only want to enable the package if the VideoPress module is active. any ideas on this?

Reply:
So when initializing from the Jetpack plugin I can't use Config, but when initializing from the stand-alone plugin we might still be able to use Config